common/variant/src/main/java/org/apache/spark/types/variant/VariantUtil.java

```diff
@@ -120,6 +120,12 @@ public class VariantUtil {
   // Long string value. The content is (4-byte little-endian unsigned integer representing the
   // string size) + (size bytes of string content).
   public static final int LONG_STR = 16;
+  // year-month interval value. The content is one byte representing the start and end field values
+  // (1 bit each starting at least significant bits) and a 4-byte little-endian signed integer
```

Comment:
Similar to the README, we should document what 0 and 1 represent for the start and end fields.